src/validation.h

```diff
@@ -32,6 +32,8 @@
 #include <boost/unordered_map.hpp>
 #include <boost/filesystem/path.hpp>
 
+#include <univalue.h>
```

Comment:
Preferably dont add a UniValue dependancy here...maybe just define a struct which carries the info you want?